File: Examples/CSharp/ProgrammersGuide/WorkingWithProjects/WriteFiscalYearProperties.cs
using System.IO;
using Aspose.Tasks;
using Aspose.Tasks.Saving;

namespace Aspose.Tasks.Examples.CSharp.WorkingWithProjects
{
    public class WriteFiscalYearProperties
    {
        public static void Run()
        {
            //ExStart: WriteFiscalYearProperties
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_Projects();

            //Create a project instance
            Project prj = new Project(dataDir+ "WriteFiscalYearProperties.mpp");
            //Set fiscal year properties
            prj.Set(Prj.FyStartDate, Month.July);
            prj.Set(Prj.FiscalYearStart, true);
            prj.Save(dataDir+ "saved.mpp", SaveFileFormat.MPP);
            //ExEnd: WriteFiscalYearProperties
        }
    }
}

File: Examples/VisualBasic/HandlingBasicProperties/WriteFiscalYearProperties.vb
Imports Microsoft.VisualBasic
Imports System.IO
Imports Aspose.Tasks
Imports Aspose.Tasks.Saving

Namespace Aspose.Tasks.Examples.VisualBasic.HandlingBasicProperties
    Public Class WriteFiscalYearProperties
        Public Shared Sub Run()
            'ExStart: WriteFiscalYearProperties
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_HandlingBasicProperties()

            ''Create a project instance
            'Dim prj As New Project(dataDir & "WriteFiscalYearProperties.mpp")
            ''Set fiscal year properties
            '         prj.[Set](prj.FyStartDate, Month.July)
            '         prj.[Set](prj.FiscalYearStart, True)
            '         prj.Save(dataDir & "saved.mpp", SaveFileFormat.MPP)
            'ExEnd: WriteFiscalYearProperties
        End Sub
    End Class
End Namespace